packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -136,6 +136,7 @@ export interface Field<T = any, V = Vector<T>> {
    */
   config: FieldConfig;
   values: V; // The raw field values
+  additionalValues?: Record<string, V>;
```

Comment:
personally not a huge fan of long prop names, maybe just `values2` even if just to keep them adjacent in code/debugger due to same prefix?

i still think that just having `values: [1678387711084], values2: [0.000001]` or `values: [1678387711084], fracts: [0.000001]` is something to consider. it keeps the units uniform for the whole field and removes an extra unit-defining `nanos` layer/key.

Reply:
hi @leeoniya , will work on a better fieldName, 👍 

regarding the fractions, i totally see your point, but.. i tested it, and the floating-point numbers do not roundtrip unfortunately 😿 . try this:
```js
>>> console.log(1000000*0.000123)
123.00000000000001
```
and i need to do something like this because i need to get to a "whole nanosecond-number in a string" representation.

Reply:
you can Math.round() that result, but it looks like we're moving forward to int `nanos`, so probably no longer relevant.